src/api/attributes.ts

```
@@ -63,7 +63,15 @@ function getAttr(
 
   if (Object.hasOwn(elem.attribs, name)) {
     // Get the (decoded) attribute
-    return !xmlMode && rboolean.test(name) ? name : elem.attribs[name];
+    const value = elem.attribs[name];
+
+    // Boolean attributes normally return the attribute name, but `hidden`
+    // accepts the special value `until-found` per HTML spec.
```

ESLint

[error] Expected a block comment instead of consecutive line comments.

(multiline-comment-style)